core/backend/src/ChangeSummaryManager.ts

```diff
@@ -525,7 +527,10 @@ export class ChangeSummaryManager {
       if (stat.error && stat.error.status !== DbResult.BE_SQLITE_OK)
         throw new IModelError(stat.error.status, stat.error.message);
 
-      changeSummaryId = stat.result!;
```

Comment:
None of this is right. The return type for `extractChangeSummary` should be changed to indicate that either `error` or `result` will be defined.